test/integration/create-from-url/createFromURLTemplate.spec.js

```diff
@@ -6,70 +6,126 @@ var page = require('./createFromURL.js');
 
 describe('authenticated e2e-user', function() {
 
-  beforeEach(function() {
+  var project = h.projectDetails();
+
+  beforeAll(function() {
     h.commonSetup();
     h.login();
   });
 
-  afterEach(function() {
-    h.commonTeardown();
+  afterAll(function() {
+    h.deleteProject(project);
+    h.afterAllTeardown();
+  });
+
+  describe('using an image stream and image tag supplied as query string params', function() {
+    var qs = '?imageStream=nodejs&imageTag=4';
+    it('should display details about the the image', function() {
+      page.visit(qs);
+      expect(element(by.css('h1')).getText()).toEqual('Node.js 4');
+    });
+    it('should create a project to add to', function(){
```

Comment:
A quick thoughts here. an `it()` should have an `expect()` inside as an assertion of something, otherwise its not quite a test.  After the `tab.click()` we can probably `browser.wait()` then check that the url contains the project name?